app/views/home/about.html.erb

```diff
@@ -341,6 +342,22 @@
           <p>Astronomer exploring the role of data science in academia at <a href="http://escience.washington.edu/">University of Washington's eScience Institute</a>. Core contributor to <a href="http://scikit-learn.org/">scikit-learn</a> and other science-focused Python packages.</p>
         </div>
 
+        <div class="main" id="business_model">
+          <h1>Business Model</h1>
+          <p>The Journal of Open Source Software is an open access journal <em>committed to running at minimal costs, with zero publication fees (<a href="https://en.wikipedia.org/wiki/Article_processing_charge">article processing charges</a>) or subscription fees</em>.
+            With volunteer effort from our editorial board, community reviewers, donations and minimal infrastructure costs we believe JOSS can remain a free community service.</p>
+
+          <p>In the spirit of transparency, below is an outline of our current running costs:</p>
+
+          <ul>
+            <li>Annual Crossref <a href="http://www.crossref.org/02publishers/20pub_fees.html">membership</a>: $275 / year</li>
+            <li>JOSS paper DOIs: $1 / accepted paper</li>
+            <li>JOSS website hosting (Heroku): $19 / month</li>
+          </ul>
```

Comment:
Yeah... let me look back into whether what we did for SCF could be translated to this context.